tests/core/test_config.py

```diff
@@ -11,7 +11,7 @@ def test_get_yaml(project_path: Path):
         file=project_path / "resources/application.yaml"
     )
 
-    assert config.security.admin_pwd == "admin"
+    assert config.security.admin_pwd == ""
```

Comment:
I don't see the point of such a unit test. At best, this test validates that the file is in YAML format, but PyCharm can already do that. It would be better to remove it entirely.